src/macros.js

```diff
@@ -65,6 +65,9 @@ export default function adMacroReplacement(string, uriEncode, customMacros) {
   macros['{mediainfo.name}'] = this.mediainfo ? this.mediainfo.name : '';
   macros['{mediainfo.duration}'] = this.mediainfo ? this.mediainfo.duration : '';
   macros['{player.duration}'] = this.duration();
+  macros['{player.pageUrl}'] = (window.location !== window.parent.location) ?
```

Comment:
Maybe a comment here explaining that this conditional is used to detect iframes? It may be unfamiliar to some.

Reply:
I think that secure iframes throw if you access parent location, we'd want to double check and wrap in a try/catch if so

Reply:
Videojs has a isInIframe helper on the Dom until object

Reply:
on chrome/ff/safari, `window.parent.location` doesn't throw, although `window.parent.location.href` does.

Reply:
didn't realise we had isInFrame. will use that instead then.